nucypher-core/src/dkg.rs

```diff
@@ -277,8 +277,10 @@ pub mod session {
         }
     }
 
-    type SessionSecretFactorySeedSize = U32; // the size of the seed material for key derivation
-    type SessionSecretFactoryDerivedKeySize = U32; // the size of the derived key
+    type SessionSecretFactorySeedSize = U32;
+    // the size of the seed material for key derivation
+    type SessionSecretFactoryDerivedKeySize = U32;
+    // the size of the derived key
```

Comment:
IMHO the comments should be placed above the line and not under the line they refer. (this seems like a formatting tool thing)

```suggestion
    // the size of the seed material for key derivation
    type SessionSecretFactorySeedSize = U32;
    // the size of the derived key
    type SessionSecretFactoryDerivedKeySize = U32;
```

Reply:
`cargo fmt` is messing these up constantly for me, I may need to edit formatting rules.